java/food-ordering/app/restate-app/src/main/java/dev/restate/sdk/examples/types/StatusEnum.java

```diff
@@ -1,6 +1,6 @@
 package dev.restate.sdk.examples.types;
 
-public enum Status {
+public enum StatusEnum {
```

Comment:
Why the rename to `Enum`?

Reply:
Because it collided with Status in the proto definitions